README.md

```diff
@@ -51,6 +51,35 @@ jobs:
           args: --all-features
 ```
 
+### With `args-file`
+```yaml
+name: Clippy check
+jobs:
+  clippy_check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          components: clippy
+          override: true
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args-file: "./clippy_args.sh"
```

Comment:
Maybe the extension should be e.g. `.txt` since it's not an executable shell script?